src/Utilities/DownloadPlaybookButton.js

```diff
@@ -68,11 +69,16 @@ const DownloadPlaybookButton = ({ isDisabled, rules, systems }) => {
   return (
     <Button
       key="playbook-download-button"
-      variant="primary"
+      variant="secondary"
       isDisabled={isDisabled}
       onClick={() => download(preparePayload(rules, systems))}
     >
-      {intl.formatMessage(messages.downloadPlaybookButtonText)}
+      <span>
+        <Icon>
+          <DownloadIcon />
+        </Icon>{' '}
```

Comment:
Prefer to use CSS for spacing rather than spaces.

Reply:
Done!